docker/Dockerfile.umn

```diff
@@ -18,8 +18,16 @@ RUN apt-get update  && apt-get install -y \
     ros-humble-grid-map-rviz-plugin \
     ros-humble-can-msgs \
     ros-humble-usb-cam \
+    python3-pytest \
+    python3-opencv \
+    python3-serial \
+    python3-numpy \
+    libboost-all-dev \
```

Comment:
I'm curious what this depends on it's quite bulky, is it can?

Reply:
Yes, the CAN bridge depends on libboost-all-dev